js/pages/tola_management_pages/user/components/edit_user_programs.js

```diff
@@ -33,10 +32,10 @@ const create_program_objects = (programs, store) => Object.entries(programs)
                                                            }),{})
 
 //we need to flatten the country -> program heirarchy to support the virtualized table
-const flattened_listing = (countries, programs) => countries.flatMap(country =>
+const flattened_listing = (countries, programs, isExpanded) => countries.flatMap(country =>
                                                         [
                                                             country,
-                                                            ...(country.expanded ? Array.from(country.programs)
+                                                            ...(isExpanded(country.id) ? Array.from(country.programs)
```

Comment:
for my edification: I am not familiar with this three-dot syntax. What is it?

Reply:
Spread syntax - the return there is an array, so this spreads the elements out into a flat array of [country, program1, program2, etc.] (given the country.programs array of [program1, program2, etc.])